src/machines/workflow_machines.rs

```diff
@@ -591,6 +605,10 @@ impl WorkflowMachines {
                     let canm = self.add_new_machine(continue_as_new(attrs));
                     self.current_wf_task_commands.push_back(canm);
                 }
+                WFCommand::AckCancelled(attrs) => {
```

Comment:
Didn't we rename this?

Reply:
This is a core-internal name but yeah I'll keep it consistent